test_aodndata/moorings/test_mooringsHandler.py

```diff
@@ -164,27 +166,28 @@ def test_all_formats_zip(self):
             self.assertTrue(f.is_harvested)
 
     def test_burst_processing(self):
-        handler = self.run_handler(BURST_NC,
-                                   include_regexes=['IMOS_ANMN-NRS_.*\\.nc'],
-                                   check_params={'checks': ['cf', 'imos:1.4']}
+        zip_file = make_zip(self.temp_dir, [BURST_NC, BURST_ADCP])
+        handler = self.run_handler(zip_file,
+                                   include_regexes=['.*\\.nc'],
+                                   check_params={'checks': ['cf', 'imos:1.4'],
+                                                 'skip_checks': ['check_dimension_order']}
                                    )
 
-        self.assertEqual(2, len(handler.file_collection))
+        self.assertEqual(3, len(handler.file_collection))
 
         f = handler.file_collection[0]
         self.assertEqual(f.name, BURST_NC_BASENAME)
         self.assertEqual(f.dest_path, os.path.join('IMOS/ANMN/NRS/NRSROT/Biogeochem_timeseries', BURST_NC_BASENAME))
-        self.assertTrue(f.is_checked)
-        self.assertTrue(f.is_stored)
-        self.assertTrue(f.is_harvested)
 
         # there should also be a burst-averaged file
-        f = handler.file_collection[1]
+        f = handler.file_collection[-1]
```

Comment:
wouldn't it make more sense to use filtering option to get the file you need without using a specific index?

Reply:
I could, but this is simpler. Also, I know that the one I want is the last file added to the collection, because the burst-averaging happens after the resolve step. So if the last file in the collection _isn't_ the burst-averaged file, then there is something wrong, and the unittest _should_ fail.